challenges/objects-arrays.js

```diff
@@ -83,37 +128,47 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Hawk-eagle, crowned","population":10,"scientific_name":"Spizaetus coronatus","state":"Florida"},
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
-/* Request 1: .forEach()
+ /* Request 1: .forEach()
 
-The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
+ The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
-console.log(animalNames);
+
+ const animalNames = [animal_name + scientific_name];
+
+ zooAnimals.forEach(zooAnimals => console.log(animalNames));
+
+ console.log(animalNames);
 
 /* Request 2: .map()    
 
-The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
+ The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
-*/
+ */
 
-const lowerCase = [];
-console.log(lowerCase); 
+ const lowerCase = zooAnimals.map(zooAnimals, animal_name.toLowerCase);
 
-/* Request 3: .filter() 
+ console.log(lowerCase); 
 
-The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
+ /* Request 3: .filter() 
 
-*/
-const largerPopulation = [];
+ The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
+
```

Comment:
This is pretty close but it is still psuedocode 
*function getAnimalsLessThanFive() {
   var animals = zooAnimals.filter(animal => animalspopulation < 6);
   return animals;
 }*